.github/bin/split.bash

```diff
@@ -92,10 +88,11 @@ check_assets() {
 
   if [[ ${package} == "" || ${package} == "storefront" ]]; then
     stat -t $(storefront_assets_list) > /dev/null
+    check_admin_assets
```

Comment:
This looks odd. Are there any storefront-specific assets we can check for? If not, I'd suggest removing this whole conditional branch altogether.

Reply:
yes, there's a storefront admin module. the other check is still there because the storefront itself is still build with webpack. But vite generates random files us far as I can tell, so this is just a basic smoke test

Reply:
you could check for the .vite/manifest.json file if that exists. that would guarantee vite successfully built it